dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -1435,12 +1436,20 @@ public Identifier getRelatedIdentifier(Contentlet contentlet,String relationship
         return permissionAPI.filterCollection(contentFactory.getRelatedLinks(contentlet), PermissionAPI.PERMISSION_READ, respectFrontendRoles, user);
     }
 
-    @CloseDBIfOpened
+
+    @Deprecated
     @Override
     public List<Contentlet> filterRelatedContent(Contentlet contentlet, Relationship rel,
             User user, boolean respectFrontendRoles, Boolean pullByParent, int limit, int offset,
             String sortBy)
             throws DotDataException, DotSecurityException {
+        return filterRelatedContent(contentlet, rel, user, respectFrontendRoles, pullByParent, limit, offset);
+    }
+
+    @CloseDBIfOpened
+    public List<Contentlet> filterRelatedContent(Contentlet contentlet, Relationship rel,
```

Comment:
This should not be public

Reply:
The access modifier was changed to default and the method annotated as @VisibleForTesting